src/platform/packages/shared/kbn-workflows/types/v1.ts

```diff
@@ -135,7 +136,7 @@ export interface EsWorkflowSchema {
   id: string;
   name: string;
   description: string;
-  status: WorkflowStatus;
+  status: ExecutionStatus;
```

Comment:
why this change? workflow status is meant to reflect state of the workflow itself such as "active", "draft", "deleted".

Reply:
@Kiryous thinking out loud but maybe it's not status? can I have a deleted draft, for example?

Reply:
yeah, we can make adjustments, just ExecutionStatus doesn't make sense here. or it mean to be "lastExecutionStatus" @skynetigor ?

Reply:
ah, I think it's unintentional
I didn't want to change Workflow model but wanted to change WorkflowExecution model so that it has execution status